readme.md

```diff
@@ -156,6 +156,7 @@ programming languages with appropriate bindings for Ruby.
 - [ruby-dnn](https://github.com/unagiootoro/ruby-dnn) - Simple deep learning for Ruby.
 - [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch)
   using [rice](https://github.com/jasonroelofs/rice).
+- [mxnet](https://mxnet.apache.org/) - Ruby bindings for [mxnet](https://github.com/mrkn/mxnet.rb).
```

Comment:
@bkmgit , could you please exchange the links: Ruby bindings project first, upstrem project last.

Reply:
@arbox Thanks. Done